.eslintrc

```diff
@@ -270,13 +270,13 @@
         "object-curly-newline": "off",
         "object-curly-spacing": ["error", "always"],
         "object-property-newline": "off",
-        "one-var": ["error", "consecutive"],
-        "one-var-declaration-per-line": "error",
+        "one-var": "off",
+        "one-var-declaration-per-line": "off",
         "operator-assignment": "error",
         "operator-linebreak": ["error", "after"],
         "padded-blocks": ["error", "never"],
         "padding-line-between-statements": [
-            "error",
+            "off",
             {
                 "blankLine": "always",
```

Comment:
Can this options be removed in case it is not used?

Reply:
Yes, the options can be removed, which I did. Thank you